src/ecrc-frontend/src/components/page/consent/Consent.js

```diff
@@ -138,7 +138,6 @@ export default function Consent({
         error.request.response &&
         JSON.parse(error.request.response)
       ) {
-        setToError(true);
```

Comment:
why removed?

Reply:
If you expand it, the value is already set in line 134